extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -30,8 +30,14 @@ class AmpAccordion extends AMP.BaseElement {
     /** @const @private {!NodeList} */
     this.sections_ = this.getRealChildren();
 
+    /** @const @private */
+    this.id_ = this.element.id;
+
     this.element.setAttribute('role', 'tablist');
     const boundOnHeaderClick_ = this.onHeaderClick_.bind(this);
+    const accordionSession_ = sessionStorage.getItem(this.id_);
```

Comment:
Also please add comments that describes what is the format for the key and value.